packages/dashboard/examples/ed-visits-county-file.json

```diff
@@ -1,712 +1,618 @@
+        "values": [
+          "United States",
+          "Alabama",
+          "Alaska",
+          "Arizona",
+          "Arkansas",
+          "California",
+          "Colorado",
+          "Commonwealth of the Northern Mariana Islands",
+          "Connecticut",
+          "Delaware",
+          "District of Columbia",
+          "Florida",
+          "Georgia",
+          "Hawaii",
+          "Idaho",
+          "Illinois",
+          "Indiana",
+          "Iowa",
+          "Kansas",
+          "Kentucky",
+          "Louisiana",
+          "Maine",
+          "Maryland",
+          "Massachusetts",
+          "Michigan",
+          "Minnesota",
+          "Mississippi",
+          "Missouri",
+          "Montana",
+          "Nebraska",
+          "Nevada",
+          "New Hampshire",
+          "New Jersey",
+          "New Mexico",
+          "New York",
+          "New York City",
+          "North Carolina",
+          "North Dakota",
+          "Ohio",
+          "Oklahoma",
+          "Oregon",
+          "Pennsylvania",
+          "Puerto Rico",
+          "Rhode Island",
+          "South Carolina",
+          "South Dakota",
+          "Tennessee",
+          "Texas",
+          "Utah",
+          "Vermont",
+          "Virgin Islands",
+          "Virginia",
+          "Washington",
+          "West Virginia",
+          "Wisconsin",
+          "Wyoming"
         ],
-        [
-            {
-                "width": 4,
-                "widget": "filtered-text1701791173259"
-            },
-            {
-                "equalHeight": false,
-                "width": 4,
-                "widget": "filtered-text1701791674571"
-            },
-            {
-                "width": 4,
-                "widget": "filtered-text1701791683990"
-            },
-            {
-                "width": null
-            }
-        ]
+        "active": "United States",
+        "tier": 1,
+        "datasetKey": "resp-data.json",
+        "filterBy": "File Name",
+        "fileName": "NSSPSubState${query}",
+        "apiFilter": {
+          "apiEndpoint": "https://www.cdc.gov/wcms/vizdata/Respitory_Viruses/NSSP%20Subsets/NSSPGeographyReference.json",
+          "valueSelector": "geography",
+          "textSelector": "geography",
+          "defaultValue": "United States",
+          "autoLoad": true
+        },
+        "whitespaceReplacement": "Remove Spaces"
+      },
+      {
+        "key": "County",
+        "type": "datafilter",
+        "columnName": "county",
+        "showDropdown": true,
+        "usedBy": ["chart1701791122773", "filtered-text1701791173259", "filtered-text1701791674571", "filtered-text1701791683990", "filter-dropdowns1701792379133", "filtered-text1701888972205", "filtered-text1701888980655"],
+        "values": ["All"],
+        "active": "All",
+        "tier": 2,
+        "parents": ["State"]
```

Comment:
@mpallansch I just caught that this also needed to be changed in the config to State, it was still set to Dashboard 3. 

I made a couple other changes as well.